source/features/improve-shortcut-help.js

```diff
@@ -35,76 +36,65 @@ export function registerShortcut(groupId, hotkey, description) {
 	shortcuts.set(hotkey, {groupId, hotkey, description});
 }
 
-function splitKeys(keys) {
-	return keys.replace(/\S+/g, '<kbd>$&</kbd>');
-}
-
-function improveShortcutHelp() {
-	// Remove redundant "Show All" button
-	select('.js-see-all-keyboard-shortcuts').remove();
-
-	// Close the modal with Esc
-	select('.js-facebox-close').dataset.hotkey = 'Escape';
-
-	const modal = select('.shortcuts');
-	const groupElements = select.all('tbody', modal);
-	for (const groupElement of groupElements) {
-		const groupTitle = select('tr th:nth-child(2)', groupElement).textContent;
-		if (groupTitle in groups) {
-			const groupId = groups[groupTitle];
+const splitKeys = keys => keys.replace(/\S+/g, '<kbd>$&</kbd>');
 
-			if (groupElement.style.display === 'none') {
-				if (groupId.startsWith('project') && !isProject()) {
-					// The "Projects"-related groups are quite big and not interesting to most users
-					groupElement.remove();
-				} else {
-					// Reduce opacity of previously hidden groups
-					groupElement.removeAttribute('style');
-					groupElement.classList.remove('js-hidden-pane');
-					groupElement.classList.add('rgh-inactive-shortcut-group');
-				}
-			} else {
-				// Push relevant (not hidden) groups to the top
-				groupElement.parentElement.prepend(groupElement);
-			}
+const fixKeys = dialog => {
+	for (const key of select.all('kbd', dialog)) {
+		if (key.textContent.includes(' ')) {
+			key.replaceWith(domify(splitKeys(key.textContent)));
+		}
+	}
```

Comment:
No need to change the type of functions, this diff could be halved 😃

Reply:
I just started writing in my normal style and then realised the other was a named function so I adjusted it to my style.
Do you have a preference here? I can totally adjust to that, but e.g. the `splitKey` I like more as a arrow function.

Reply:
Unless they are one-liners I prefer function declarations at the top level simply because the first word of the line is the obvious `function` instead of the generic `const`. But also declared functions have names that appear while debugging and arrow functions are nameless (even though we’re naming their `const`)
Lastly, function declarations don’t need the semicolon after the closing curly brace 😁

Reply:
> But also declared functions have names that appear while debugging and arrow functions are nameless (even though we’re naming their `const`)

That's no longer entirely correct. Since ES2015, JS engines infer the function name from the variable name: http://2ality.com/2015/09/function-names-es6.html